styles/components/_personal-bio.scss

```diff
@@ -4,6 +4,7 @@
 
   letter-spacing: -0.02em;
   width: 100%;
+  margin-top: space("xl");
```

Comment:
For FunFacts, PastClients, and PersonalBio we could similarly have the largest, Figma margin here, too.
```css
  @media (min-width: $lg-screen-width) {
    margin-top: space("3xl");
  }
```